lib/backend/k8s/resources/profile.go

```diff
@@ -135,11 +227,22 @@ func (c *profileClient) EnsureInitialized() error {
 }
 
 func (c *profileClient) Watch(ctx context.Context, list model.ListInterface, revision string) (api.WatchInterface, error) {
-	if len(list.(model.ResourceListOptions).Name) != 0 {
+	resl := list.(model.ResourceListOptions)
+	if len(resl.Name) != 0 {
 		return nil, fmt.Errorf("cannot watch specific resource instance: %s", list.(model.ResourceListOptions).Name)
 	}
 
-	k8sWatch, err := c.clientSet.CoreV1().Namespaces().Watch(metav1.ListOptions{ResourceVersion: revision})
+	nsRev, saRev, err := c.splitRev(revision)
+	if err != nil {
+		return nil, err
+	}
+
+	log.WithFields(log.Fields{
+		"NSPROFRev": nsRev,
```

Comment:
nit: `NSNPROFRev` is a bit hard to parse - let's just use the name of the variable `nsRev` and `saRev`